plugins/hangmanplugin.py

```diff
@@ -45,6 +45,9 @@ def startGame(self, ctx, msg):
                     self.guessedLetters = []
                     self.misses = 0
                     ctx.msg("{:s} has given me a word. Try guessing some letters! (!guess <letter>)".format(msg.player))
+                    Legitimate = checkWord(self.word)
```

Comment:
Two things here:
* `checkword()` is a method of this class, so must be called via `self.checkword()`
* There is no need for a temporary variable here, the whole thing can be done like this: `if not self.checkword(self.word):`